benches/time_timeout.rs

```diff
@@ -0,0 +1,62 @@
+//! Benchmark spawning a task onto the basic and threaded Tokio executors.
+//! This essentially measure the time to enqueue a task in the local and remote
+//! case.
+
+use std::time::{Duration, Instant};
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use tokio::time::timeout;
+
+// a vevry quick async task, but might timeout
+async fn quick_job() -> usize {
+    1
+}
+
+fn single_thread_scheduler_timeout(c: &mut Criterion) {
+    do_test(c, 1, "single_thread_timeout");
+}
```

Comment:
This is using a multi-threaded runtime with one thread. It would also be nice to include the current-thread runtime.